CHANGELOG.md

```diff
@@ -18,14 +26,33 @@ Branch: `nilrt/master/hardknott`
 - [Upgraded](https://github.com/ni/nilrt/pull/73) the nilrt-build dockerfile to a debian 10 base.
   - Added a more user-friendly way to enter the docker build container using `docker-compose`.
 
+#### Fixed
+- [Fixed](https://github.com/ni/meta-nilrt/pull/308) an issue where SSH sessions were not properly closed on reboot or shutdown.
+
+#### Removed
+- Removed the following OpenEmbedded Layers either because they were abandoned upstream, or because official support was dropped within NILRT.
+  - [meta-ivi](https://github.com/ni/nilrt/pull/119).
+  - [meta-java](https://github.com/ni/nilrt/pull/120).
+  - [meta-measured](https://github.com/ni/nilrt/pull/127).
+  - [meta-mingw](https://github.com/ni/nilrt/pull/124).
+  - [meta-mono](https://github.com/ni/nilrt/pull/123).
+
 
 ### meta-nilrt
 
 #### Changed
 - `gcc` has been upgraded to version `10.2`.
```

Comment:
I suppose you'll add the release branch hotfixes in an upcoming PR?

Reply:
Per Alex, that will be part of going final/release tasks.